### PR TITLE
GSoC: Add Proposal for KeptnTasks Repository

### PR DESCRIPTION
**Pull request by agardnerIT**

### CODE REVIEW

File: summerofcode/2023.md [hunk 1]
```diff
@@ -242,6 +242,24 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
...
+
+- Notifying Chat tools
+- Triggering external tools (eg. a pipeline)
+- Integrations with Systems of Record (i.e. to update a CMDB)
+
+This project lends itself to GSoC due to the modular nature of the tasks which almost guarantees that something will be delivered - the task delivery can be planned an organised easily into sprints across the project lifecycle. It is also relatively easy - given that the tasks are simply JavaScript - it should be easy for most first time contributors.
+
+- Expected outcome: A catalogue of useful, re-usable `KeptnTaskDefinitions` that will serve as useful reference material for adopters
+- Recommended Skills: Typescript and / or JavaScript, basic understanding of Keptn Lifecycle Toolkit, Kubernetes basic knowledge, knowledge of CRDs
+- Mentor[s]: [Adam Gardner (adam at agardner dot net)](https://github.com/agardnerit)
```
Comment (aliok) [inline summerofcode/2023.md:258]: @agardnerIT Can you ask one of the people listed in https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L512 to LGTM this PR? Your name is not in that file and CNCF org admins require the "blessing" of the maintainers from that list.

Comment (thisthat) [reply summerofcode/2023.md:258]: Hey @aliok, I spoke with @agardnerIT and LGTM :)

**Review by thisthat — APPROVED** [review]
LGTM

**Comment by aliok** [comment]
LGTM

**Comment by DavidPHirsch** [comment]
@aliok @nate-double-u looks like this is still stuck.. are there any additional changes needed?

**Comment by aliok** [comment]
IMO it is good. cc @nate-double-u

**Review by nate-double-u — APPROVED** [review]
Thanks for this @agardnerIT, @thisthat & @DavidPHirsch. And thanks for the review help @aliok!

**Pull request merged**